File: python/PetShop.py
class PetShop:
    # Constructor
    def __init__(self, id, nama_produk, harga_produk, stok_produk):
        self.__id = id
        self.__nama_produk = nama_produk
        self.__harga_produk = harga_produk
        self.__stok_produk = stok_produk
    
    # Getter methods
    def get_id(self):
        return self.__id
    
    def get_nama_produk(self):
        return self.__nama_produk
    
    def get_harga_produk(self):
        return self.__harga_produk
    
    def get_stok_produk(self):
        return self.__stok_produk
    
    # Setter methods
    def set_id(self, id):
        self.__id = id
    
    def set_nama_produk(self, nama_produk):
        self.__nama_produk = nama_produk
    
    def set_harga_produk(self, harga_produk):
        self.__harga_produk = harga_produk
    
    def set_stok_produk(self, stok_produk):
        self.__stok_produk = stok_produk

File: java/PetShop.java

public class PetShop {
    // Attributes
    private int id;
    private String nama_produk;
    private int harga_produk;
    private int stok_produk;
    
    // Constructor
    public PetShop(int id, String nama_produk, int harga_produk, int stok_produk) {
        this.id = id;
        this.nama_produk = nama_produk;
        this.harga_produk = harga_produk;
        this.stok_produk = stok_produk;
    }
    
    // Getter methods
    public int getId() {
        return id;
    }
    
    public String getNamaProduk() {
        return nama_produk;
    }
    
    public int getHargaProduk() {
        return harga_produk;
    }
    
    public int getStokProduk() {
        return stok_produk;
    }
    
    // Setter methods
    public void setId(int id) {
        this.id = id;
    }
    
    public void setNamaProduk(String nama_produk) {
        this.nama_produk = nama_produk;
    }
    
    public void setHargaProduk(int harga_produk) {
        this.harga_produk = harga_produk;
    }
    
    public void setStokProduk(int stok_produk) {
        this.stok_produk = stok_produk;
    }
}